libpod/container_internal.go

```diff
@@ -924,6 +932,26 @@ func (c *Container) cleanup(ctx context.Context) error {
 	return lastError
 }
 
+// cleanupVolumeDependencies removes the ID of the container that is being removed from the list of
+// container dependencies of the named volumes it is using
+func (c *Container) cleanupVolumeDependencies(ctx context.Context) error {
+	for _, vol := range c.config.Spec.Mounts {
+		fmt.Println(vol)
```

Comment:
This looks like a debug statement maybe?

Reply:
Yes it is, thanks for pointing it out. Fixed